packages/urls/src/urls.ts

```diff
@@ -104,6 +104,11 @@ export const CHROME_ANDROID_URL =
     'https://play.google.com/store/apps/details?id=com.android.chrome';
 export const TOR_PROJECT_URL = 'https://www.torproject.org/';
 export const TOR_SNOWFLAKE_PROJECT_URL = 'https://snowflake.torproject.org/';
+export const TOR_SNOWFLAKE_KB_URL = 'https://trezor.io/learn/a/tor-snowflake-in-trezor-suite';
```

Comment:
not working

Reply:
1. FYI the urls does not exist at this moment, so it will return you error. It will work once KB articles will be published.